SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.cc

```diff
@@ -120,6 +128,24 @@ void SiPixelDigitizerAlgorithm::init(const edm::EventSetup& es) {
   //gets the map and geometry from the DB (to kill ROCs)
   es.get<SiPixelFedCablingMapRcd>().get(map_);
   es.get<TrackerDigiGeometryRecord>().get(geom_);
+
+  // Read template files for charge reweighting
+  if (UseTemplateAgeing){
```

Comment:
@schuetzepaul 
I wonder if this is the best name for this feature.  The reweighting by data/MC templates simply makes the cluster shape more realistic.  It can be used even if there is little or no radiation damage...  Just a thought.

Reply:
You're completely right. I took this flag from Andreas and never touched it. Probably something like `UseTemplates` or `UseReweighting` would describe the feature more adequately.

Reply:
Changed the name / config parameter to `UseReweighting`.